app/models/package_manager/pypi/version_processor.rb

```diff
@@ -11,20 +11,31 @@ def initialize(project_releases:, project_name:, known_versions:)
 
       def execute
         @project_releases.map do |project_release|
```

Comment:
The case we're missing here is if libraries knows about a version but the api doesn't (i.e. the version number is not in `@project_releases`)